### PR TITLE
[ENH] test parameters for RocketRegressor

### PR DESCRIPTION
**Pull request by iaryangoyal**
Reference Issues/PRs
Issue - https://github.com/sktime/sktime/issues/3429

Please use keywords (e.g., Fixes) to create https://github.com/sktime/sktime/issues/3429 to the issues or pull requests
you resolved, so that they will automatically be closed when your pull request
is merged. See https://github.com/blog/1506-closing-issues-via-pull-requests.
If no issue exists, you can open one here: https://github.com/sktime/sktime/issues

What does this implement/fix? Explain your changes.
I've added another dictionary to the list returned by the method, where num_kernels is set to a different value (in this case, 100). This provides a second parameter set for testing the RocketRegressor estimator.

### CODE REVIEW

**Comment by iaryangoyal** [comment]
@fkiraly Please review this of adding two test parameters in RocketRegressor and merge the same

**Review by fkiraly — CHANGES_REQUESTED** [review]
Sure. Kindly address my review comments on the added test parameters from the other PR.

**Comment by iaryangoyal** [comment]
Sure sir,
talking about review comments on other PR first is Some requests:

1. can you cover the other parameters too? (except n_jobs, random_state)
    I will work on them to see if this PR is correct to merge in the main or not this will help me.
2. there is no need for the "if", the test framework interprets condition-less returns as the default.
    I have removed the "if" statement and returned the two test parameters directly.

**Comment by fkiraly** [comment]
Could you also kindly resolve the conflict in the file?

**Comment by iaryangoyal** [comment]
> Could you also kindly resolve the conflict in the file?

I have solved the conflict in the file

File: sktime/regression/kernel_based/_rocket_regressor.py [hunk 1]
```diff
@@ -203,8 +203,7 @@ def _get_delegate(self):
 
     @classmethod
     def get_test_params(cls, parameter_set="default"):
-        """Return testing parameter settings for the estimator.
-
+        """Return testing parameter settings for the estimator
```
Comment (yarnabrina) [inline sktime/regression/kernel_based/_rocket_regressor.py:206]: Why are we removing the period at line end and the new line? Did you check pre-commit locally? Are you not getting errors from pydocstyle?

File: sktime/regression/kernel_based/_rocket_regressor.py [hunk 2]
```diff
@@ -220,4 +219,6 @@ def get_test_params(cls, parameter_set="default"):
             instance.
             ``create_test_instance`` uses the first (or only) dictionary in ``params``.
         """
-        return {"num_kernels": 20}
+        return [{"num_kernels": 20}, {"num_kernels": 100}]
+
+
```
Comment (yarnabrina) [inline sktime/regression/kernel_based/_rocket_regressor.py:224]: Usually, it's recommended to have one empty line at end. It seems now there are 3: one was already there plus you are adding two more.

I'm not sure if your local pre-commit setup is correct. Can you please follow the setup and run the following command against your branch?

```
pre-commit run --all-files
```

It's very likely going to modify the file you changed. If it modifies other files too, please ignore those changes and undo before staging and committing this specific file.

**Review by yarnabrina — CHANGES_REQUESTED** [review]

File: sktime/regression/kernel_based/_rocket_regressor.py [hunk 3]
```diff
@@ -220,4 +219,6 @@ def get_test_params(cls, parameter_set="default"):
             instance.
             ``create_test_instance`` uses the first (or only) dictionary in ``params``.
         """
-        return {"num_kernels": 20}
+        return [{"num_kernels": 20}, {"num_kernels": 100}]
```
[the remark this answers is not in the document]
Comment (fkiraly) [reply sktime/regression/kernel_based/_rocket_regressor.py:222]: (yes)

**Comment by fkiraly** [comment]
I fixed the linting issues so we can merge this.

**Pull request merged**